Semantic versioning?

Hey, can you please use semantic versioning on this project? Using the latest version from Github Actions looks like this

```yaml
- name: Chromatic Action
  uses: chromaui/action@v1
```

Using semantic versioning would prevent introducing breaking changes to people's builds. For example, you've just released a new version which introduced some breaking changes and causes my whole CI pipeline to fail.

The error I see in Github Actions

```
Run chromaui/action@v1
  with:
    buildScriptName: storybook-build
    exitOnceUploaded: true
    exitZeroOnChanges: true
    projectToken: ***
    token: ***

Chromatic CLI v5.1.0
https://www.chromatic.com/docs/cli

Cannot read property 'scripts' of undefined
##[error]non-zero exit code
```

This is my Github Action setup that used to work before the update

```yaml
- uses: chromaui/action@v1
   with:
     buildScriptName: storybook-build
     exitOnceUploaded: true
     exitZeroOnChanges: true
     projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
     token: ${{ secrets.GITHUB_TOKEN }}
```

And `package.json`

```json
"storybook-build": "node ./scripts/storybook-build",
```

--- a/package.json
+++ b/package.json
@@ -23,6 +23,8 @@
     "@actions/core": "^1.2.4",
     "@actions/github": "^4.0.0",
     "chromatic": "^5.1.0",
+    "pkg-up": "^3.1.0",
+    "jsonfile": "^6.0.1",
     "esm": "^3.2.25"
   },
   "devDependencies": {

--- a/src/main.ts
+++ b/src/main.ts
@@ -1,6 +1,8 @@
 import { getInput, error, setFailed, setOutput } from '@actions/core';
 import { context } from "@actions/github";
 import { v4 as uuid } from 'uuid';
+import pkgUp from 'pkg-up';
+import { readFile } from 'jsonfile';
 
 import { runAll } from 'chromatic/bin/main';
 import parseArgs from 'chromatic/bin/lib/parseArgs';
@@ -63,8 +65,10 @@ async function runChromatic(options): Promise<Output> {
   const sessionId = uuid();
   const env = getEnv();
   const log = createLogger(sessionId, env);
+  const packagePath = await pkgUp(); // the user's own package.json
+  const packageJson = await readFile(packagePath);
 
-  const context = {...parseArgs([]), env, log, sessionId, flags: options} as any
+  const context = {...parseArgs([]), packagePath, packageJson, env, log, sessionId, flags: options} as any
   
   await runAll(context);  
   const { build, exitCode } = context; 

